lib/licensee/projects.rb

```diff
@@ -3,5 +3,6 @@ module Projects
     autoload :Project, 'licensee/projects/project'
     autoload :FSProject, 'licensee/projects/fs_project'
     autoload :GitProject, 'licensee/projects/git_project'
+    autoload :GithubProject, 'licensee/projects/github_project'
```

Comment:
Given that FSProject is FSProject and not FsProject, I'm wondering if this should be GitHubProject for consistency?